scene/gui/file_dialog.cpp

```diff
@@ -732,6 +724,27 @@ void FileDialog::set_current_path(const String &p_path) {
 	}
 }
 
+void FileDialog::set_root_subfolder(const String &p_root) {
+	root_subfolder = p_root;
+	ERR_FAIL_COND_MSG(!dir_access->dir_exists(p_root), "root_subfolder must be an existing sub-directory.");
```

Comment:
I wasn't sure how to solve this. If the method failed before setting `root_subfolder`, you wouldn't be able to set it from within the inspector. Now every letter spams error, but at least you can write something. But if you set invalid path, the FileDialog is technically in an invalid state.